distributed/tests/test_scheduler.py

```diff
@@ -713,7 +713,7 @@ async def test_scheduler_sees_memory_limits(s):
     await w.close()
 
 
-@gen_cluster(client=True, timeout=1000)
+@gen_cluster(client=True)
```

Comment:
This is an unrelated change which slipped in. This test usually runs a second. If we run into a timeout, I'd like to know since this may hint towards an issue

Reply:
If desired I'll remove this. I'll follow up with a few things related to worker closing/retiring later on and can include it there

Reply:
+1 for removing